### PR TITLE
Lending API improvements

### PR DESCRIPTION
**Pull request by pzaichkina**
# Description

Minor additions to the Lending API Get started page

## Type of change
- [X] New document(s)/updating existing

### CODE REVIEW

File: docs/lending/get-started.md [hunk 1]
```diff
@@ -8,6 +8,12 @@ image: "/img/banners/social/lending.png"
 import { IntegrationsList } from "@components/global/Integrations";
 import { accountingIntegrations, bankingIntegrations, commerceIntegrations } from "@components/global/Integrations/integrations";
 
+:::tip Your lending journey
+
+Our Lending API supports the data collection step of your lending journey, which starts in your own web application. Enable Lending API and configure it, then utilize our authorization component in your app. Determine where the collected data will be stored and handle the subsequent steps of the lending process in your app.
```
Comment (mcclowes) [inline docs/lending/get-started.md:13]: I don't think we've ever said 'authorization component' before? Can we say embedded link SDK or something, and add a link to the relevant page?

File: docs/lending/get-started.md [hunk 2]
```diff
@@ -79,6 +85,19 @@ In the <a href="https://app.codat.io" target="_blank">Codat Portal</a>, navigate
 
 ## Use Lending API
 
+As a prerequisite for data collection, you need to create a Codat company and connect it to a data source (for example, an accounting platform). The company represents your SMB customer's business that is sharing access to their data.
```
Comment (mcclowes) [inline docs/lending/get-started.md:88]:
```suggestion
Before you can collect your SMB customer's data, you need to create a Codat company and connect it to a data source (for example, an accounting platform). The company represents your customer's business that is sharing access to their data. 
```
Style-wise, I would prefer less formal language here

Also if we merge in the glossary PR then that might help here?

File: docs/lending/get-started.md [hunk 3]
```diff
@@ -79,6 +85,19 @@ In the <a href="https://app.codat.io" target="_blank">Codat Portal</a>, navigate
...
+As a prerequisite for data collection, you need to create a Codat company and connect it to a data source (for example, an accounting platform). The company represents your SMB customer's business that is sharing access to their data. 
+
+You can create a company:
+
+* In the [Codat Portal](https://app.codat.io) by navigating to **Companies > Create company**
+* By calling the [Create company](/lending-api#/operations/create-company) endpoint of our API
+
+Remember to [authenticate](/using-the-api/authentication) if you are making calls to our API. Navigate to **Developers > API keys** in the Portal to pick up your authorization header.
+
+To establish a connection to a data source and sync business data, your customer must grant you access. They can do so using our [Link auth flow](/auth-flow/overview) solution, which we recommend you [use in your app](/lending/get-started#authorization-flow).
```
Comment (mcclowes) [inline docs/lending/get-started.md:97]: Link auth flow - make consistent with the language early

File: docs/lending/get-started.md [hunk 4]
```diff
@@ -79,6 +85,19 @@ In the <a href="https://app.codat.io" target="_blank">Codat Portal</a>, navigate
...
+
+Once the connection is established, Codat will perform the initial syncs for the data types you have previously enabled. You can use the `New company synchronised` [webhook](/using-the-api/webhooks/core-rules-types#new-company-synchronized) to get notified once these initial syncs are complete, and at least one of them is successful.
```
Comment (mcclowes) [inline docs/lending/get-started.md:99]: Enabled isn't a thing

```suggestion
Once the connection is established, Codat will retrieve data based of your data type settings (where you have previously set 'fetch on first link'). You can use the `New company synchronised` [webhook](/using-the-api/webhooks/core-rules-types#new-company-synchronized) to get notified once these initial syncs are complete, and at least one of them is successful.
```

**Review by mcclowes — CHANGES_REQUESTED** [review]

**Review by dcoplowe — APPROVED** [review]

File: docs/lending/get-started.md [hunk 5]
```diff
@@ -36,9 +42,9 @@ Data source coverage varies by feature, so be sure to review the coverage for th
 
 ### Authorization flow
 
-As part of using the Lending API, you will need your customers to authorize your access to their data. To do so, use Link - our pre-built, conversion-optimized, and white-labelled authorization journey. 
+As part of using the Lending API, you will need your customers to authorize your access to their data. To do so, use Link - our pre-built, conversion-optimized, and white-labelled authorization flow. 
 
-You can fully embed Link in your experience by using our [Embedded Link](/auth-flow/authorize-embedded-link) component in your front-end code or choose our out-of-the-box [Hosted Link](/auth-flow/authorize-hosted-link) authorization flow to get up and running as quick as possible. 
+We recommend you fully embed the Link auth flow in your experience by using our [Embedded Link](/auth-flow/authorize-embedded-link) SDK in your front-end code. You can also choose our out-of-the-box [Hosted Link](/auth-flow/authorize-hosted-link) auth flow option to get up and running as quick as possible.
```
Comment (mcclowes) [inline docs/lending/get-started.md:47]: Front-end or frontend

**Pull request merged**